spyro/solvers/mms_acoustic.py

```diff
@@ -66,3 +69,7 @@ def analytical_solution(self, t):
         # self.analytical.assign(analytical)
 
         return self.analytical
+    
+    @override
+    def update_source_expression(self, t):
+        self.q_t.assign(2*t)
```

Comment:
I'm uncertain if this is the best practice. While using MMS for this small problem might not cause any issues, for larger cases in the future, we should prioritize wrapping scalar time-dependent variables inside a `fire.Constant` for better performance.

Reply:
I didn't get the issue here. The variable "q_t" is defined as a `Constant` is line 14. This should be faster than before because the method was returning new constants in every call. As far as I remember, using the same constant avoids some parts of JIT or assembling.

Reply:
Ah, I see now. I mistakenly thought that `q_t` was defined as a Function rather than a Constant, I didn't see line 14. Thanks for clarifying. Yes, reusing the same constant does indeed optimize the process by avoiding the overhead associated with redefining it repeatedly, leading to better performance.